include/alpaka/atomic/AtomicOaccBuiltIn.hpp

```diff
@@ -0,0 +1,387 @@
+/* Copyright 2020 Jeffrey Kelling
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#ifdef _OPENACC
```

Comment:
Shouldn't this be `ALPAKA_ACC_ANY_BT_OACC_ENABLED`?

Reply:
No, the pragma-containing files are also guarded this way in the OpenMP backends: https://github.com/alpaka-group/alpaka/blob/develop/include/alpaka/atomic/AtomicOmpBuiltIn.hpp#L12

Reply:
Ok ... maybe this is because the OMP atomics are shared between multiple backends, this is not the case for OpenACC atomic, i.e. it could be guarded by `ALPAKA_ACC_ANY_BT_OACC_ENABLED`, indeed.

Reply:
Which way of consistency is better?

Reply:
I think the current code is fine else you need an include before the guard.

Reply:
Ohh I am wrong `ALPAKA_ACC_ANY_BT_OACC_ENABLED` is coming from CMake and not from a header file. 
Then I would say `ALPAKA_ACC_ANY_BT_OACC_ENABLED` is the right way to be consistent with the other OpenACC files.